### PR TITLE
Patient upload service supports `PhoneNumberType` column in input CSV

### LINKED ISSUE #3360: Extend BulkUpload to include phone type
### Description
Bulk upoload doesnt support setting the phone type, so the user has to manually set the phone type in the UI for each student in order for them to receive a text result.

image

### Proposed solution 
Allow the csv to accept an **optional** columns for the phone number type  

column name `PhoneNumberType`
Possible values: `Mobile`, `Landline`, `Unknown` (or blank)
Unknown will leave the phone type unset (since this is optional)

Also update the bulk upload guide with the new updates and publish a new version.

### Additional context
Bulk upload doc:
https://docs.google.com/document/d/1v2jjS-BaNVsv4764GwzOxBjPEsIMSHx9TnTw2Bm2_zM/edit

Bulk upload published doc:
https://www.simplereport.gov/assets/resources/Bulk_person_upload_guide.pdf

Bulk upload runbook (internal):
https://github.com/usds/prime-simplereport-docs/blob/main/runbooks/patient-upload.md

### PR DESCRIPTION
**Pull request by nathancrtr**
## Related Issue or Background Info

- Closes #3360 

## Changes Proposed
- Get and parse `PhoneNumberType` values from input CSV and set patient phone number type  using the value

## Additional Information
- See [prime-simplereport-site PR #276](https://github.com/CDCgov/prime-simplereport-site/pull/276) for updates to bulk upload PDF

### Upload success
Screen Shot 2022-02-15 at 12 08 54 PM

* `PhoneNumberType` - `MOBILE`

Screen Shot 2022-02-15 at 12 09 10 PM

### Upload failure
Screen Shot 2022-02-15 at 12 10 06 PM

* `PhoneNumberType` - `junk`
 
Screen Shot 2022-02-15 at 12 09 57 PM

## Checklist for Author and Reviewer
- [ ] The ticket mentions supporting an `Unknown` type, but that would require some additional logic in the upload service and seems to be redundant with simply leaving the `PhoneNumberType` cell blank. If anyone feels otherwise, I can include that as an allowable value

### Infrastructure
- [x] **Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!**

### Design
- [x] Any UI/UX changes have a designer as a reviewer, and changes have been approved
- [x] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke-tested by both the engineering and design teams

### Content
- [x] Any content changes (including new error messages) have been approved by content team

### Support
- [x] Any changes that might generate new support requests have been flagged to the support team
- [x] Any changes to support infrastructure have been demo'd to support team

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [x] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

## Cloud
- [x] DevOps team has been notified if PR requires ops support
- [x] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

File: backend/src/test/java/gov/cdc/usds/simplereport/service/UploadServiceTest.java [hunk 1]
```diff
@@ -30,6 +33,7 @@
   public static final int PATIENT_PAGE_SIZE = 1000;
 
   @Autowired private PersonService personService;
+  @Autowired private PhoneNumberRepository _pnRepo;
```
Comment (zdeveloper) [inline backend/src/test/java/gov/cdc/usds/simplereport/service/UploadServiceTest.java:36]: can we name this `phoneNumberRepository`
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html#:~:text=while%20it%27s%20technically%20legal%20to%20begin%20your%20variable%27s%20name%20with%20%22_%22%2C%20this%20practice%20is%20discouraged

File: backend/src/test/java/gov/cdc/usds/simplereport/service/UploadServiceTest.java [hunk 2]
```diff
@@ -70,8 +74,14 @@ void testUploadValidCsv() {
 
     // THEN
     assertThat(getPatients()).hasSize(1);
-    assertThat(getPatients().get(0).getLastName()).isEqualTo("Best");
-    assertThat(getPatients().get(0).getAddress()).isEqualTo(address);
+    Person p = getPatients().get(0);
```
Comment (zdeveloper) [inline backend/src/test/java/gov/cdc/usds/simplereport/service/UploadServiceTest.java:77]: Thank you for adding more coverage!

**Review by zdeveloper — COMMENTED** [review]
LGTM! minor nitpick

**Review by nickclyde — APPROVED** [review]
Looks good!

**Pull request merged**